commands/unlock.js

```diff
@@ -0,0 +1,80 @@
+const util = require('../lib/util.js');
+const Discord = require('discord.js');
+
+const command = {};
+
+command.description = 'Unlock a channel';
+
+command.usage = '<global|#channel|id…> message';
+
+command.names = ['unlock'];
+
+command.execute = async (message, args, database, bot) => {
+  //Permission check
+  if (!util.isMod(message.member) && !message.member.hasPermission('MANAGE_CHANNELS')) {
+    await message.react(util.icons.error);
+    return;
+  }
+
+  let channels = await util.channelMentions(message.guild,args);
+  let embed = new Discord.MessageEmbed().setTitle('This channel has been unlocked!').setDescription(args.join(' ')).setColor(util.color.green);
+  let everyone = message.guild.roles.everyone.id;
+
+  if (channels.length) {
+    let updates = '';
+    for(let channel of channels) {
+      channel = message.guild.channels.resolve(channel);
+      if (await unlock(channel, everyone, embed))
+        updates += `<#${channel.id}>, `;
+    }
+    await message.channel.send(`Unlocked ${updates.substring(0,updates.length - 2)}!`);
```

Comment:
```suggestion
    await message.channel.send(`Unlocked ${updates.join(', ')}!`);
```

What if no channels were unlocked (updates.length === 0)?

Reply:
?

Reply:
hatte den kommentar übersehen habs gefixt